packaging/torchvision/meta.yaml

```diff
@@ -9,23 +9,27 @@ requirements:
   build:
     - {{ compiler('c') }} # [win]
     - libpng
-    - jpeg
-    - ffmpeg =4.2  # [not win]
+    - jpeg <=9b
```

Comment:
Is this related to the problem we had on the past with the size_t type? If this is causing you massive headaches we can investigate how we could use the latest.

Reply:
The jpeg pinning is required to ensure that it is installed from the Anaconda defaults channel, as opposed to conda-forge, since it could cause conflicts with libjpeg-turbo. https://github.com/pytorch/vision/pull/2818 should fix this